reana-ui/src/util.js

```diff
@@ -173,3 +174,12 @@ export function stringifyQueryParams(params) {
     skipEmptyString: true,
   });
 }
+
+/**
+ * Open Jupyter Notebook in a new tab.
+ */
+export const openJupyterNotebook = (event, sessionUri, reanaToken) => {
+  const uri = `${api}${sessionUri}?token=${reanaToken}`;
+  window.open(uri, "_blank");
```

Comment:
Completely agree that `<a>` would be nicer here. I did not use it here because of the things you mentioned in your [comment](https://github.com/reanahub/reana-ui/pull/142#discussion_r507778756). We need to stop workflow list to be clicked..
Actually only when replying to your comment I realized that both `href` and `onClick` could be used on `<a>` element :) I will change it to use `<a>`